plat/linuxu/memory.c

```diff
@@ -156,6 +161,9 @@ int ukplat_memregion_get(int i, struct ukplat_memregion_desc **m)
 	int ret;
 
 	UK_ASSERT(m);
```

Comment:
Add a newline between the `UK_ASSERT` and the `if` statement, I think it would look better.